index.html

```diff
@@ -0,0 +1,740 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="images/create-your-youtube-channel-art-and-logo.png" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+        <div class="logo-container">
+            <i class="ri-menu-line"></i>
+            <div class="logo">
+                <div class="logo-img">
+                    <img src="images/create-your-youtube-channel-art-and-logo.png" alt="">
+                </div>
+                <div id="logo-txt">YouTube</div>
+            </div>
+        </div>
+        <div class="search-container">
+            <form action="">
+                <input type="text" placeholder="Search">
+            </form>
+            <button class="search">
+                <i class="ri-search-line"></i>
+            </button>
+            <button class="mic">
+                <i class="ri-mic-fill"></i>
+            </button>
+        </div>
+        <div class="profile-container">
+            <i class="ri-video-add-line"></i>
+            <i class="ri-notification-4-line"></i>
+            <div class="profile-box">
+                <img src="images/main-qimg-217015358349186e0e382cb15c5d7c63-lq.jpeg" alt="">
+            </div>
+        </div>
```

Comment:
I suggest the Use of semantic tags (e.g., header, nav) over divs.

Reply:
- [ ] Kindly, when you give a code review, be polite and explain why you ask to change to the use of semantic tags. It would be better to say,'' I think it would be better to use a semantic tags `section`  instead of `div` to make our code clear and easy to maintain, our page accessible, and improve SEO positioning. For further detail check[ this link](https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0)" :smiley_cat: